src/lib/Containers/Gene.tsx

```diff
@@ -364,7 +364,6 @@ export default createPaginationContainer(
           aggregations {
             slice
             counts {
-              internalID
               name
```

Comment:
This is not sufficient, it needs to be replaced by `value`. I'll add some typings to the code to make sure it is flagged as required.

```suggestion
               value
               name
```